doc/adr/0009-container-image-tags.md

```diff
@@ -24,28 +24,40 @@ This ADR does not define how the image tags are used for integration tests or de
 
 ### Tagging 
 
-- libero container images shall always be tagged with ${branch}-${full_git_hash}
+- libero container images shall __always__ be tagged with both:
+  - `${branch}-${short_git_hash}-${timestamp}`
+  - `${branch}-${short_git_hash}`
+- `timestamp` format is `%Y%m%d.%H%M`
```

Comment:
I've never been sure why `branch` is useful, but `timestamp`? 😮

Reply:
TL;DR of the explanation: it provides simil-semver monotonically increasing numbers to understand whether a tag refers to a newer version.

Reply:
I understand your reaction :smile: 

Timestamp is required for renovatebot to work. It can't handle non-numeric versioning.

The thinking behind these tags is to allow existing and new repos to use renovatebot/fluxcd to track images for integration tests and deploys without forcing them to immediately implement automated semver tagging.

Reply:
Why that's needed instead of a timestamp label on the image?

Reply:
Labels aren't parsed by renovatebot and aren't recorded in compose files/charts.

Reply:
More tools doing the same thing, but none of them work well enough. 😰

Sounds like this is based on eLife's deployment requirements, so seems weird to force it on Libero.

Could you use intermediate `Dockerfile`s that just contain a `FROM`, and have Dependabot update that?

Reply:
Motivation for renovatebot comes from umbrella repos on libero side of things. The planned deployment method on eLife side can handle `branch-hash`.

By enabling the use of renovatebot we can use easily legible and locally executeable compose files for integration tests, even with component that don't use semver (minio is another culprit here).

re: dependabot - The closest I can find on its dockerfile versioning behavior is [this](https://www.rubydoc.info/github/dependabot/dependabot-core/master/Dependabot/UpdateCheckers/Docker/Docker), which looks to me like it would also require some form of numeric version.

Reply:
I would summarize this as an internal use of tagging to allow reproducible Libero components integration testing, whereas semantic versioning should remain the published tagging model for any organization that wants to rely on stable software.